### PR TITLE
app clock fixes

### PR DESCRIPTION
**Pull request by introkun**
* added Clock app to the apps Makefile section
* added Clock app sources to the Apps directory to build via toolchain

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -97,6 +97,7 @@ apps: $(CACHE)/.setup
 	@$(ECHO) $(PRINT_RECIPE)
 	@cd $(SRC_DIR)/playActivityUI && BUILD_DIR=$(BUILD_DIR)/App/PlayActivity make
 	@find $(SRC_DIR)/packageManager -depth -type d -name res -exec cp -r {}/. $(BUILD_DIR)/App/The_Onion_Installer/res/ \;
+	@cd $(SRC_DIR)/clock && BUILD_DIR=$(BUILD_DIR)/App/Clock make
```
Comment (Aemiii91) [inline Makefile:100]: This line should be `@cd $(SRC_DIR)/clock && BUILD_DIR=$(PACKAGES_APP_DEST)/Set emulated time/App/Clock make`

File: src/clock/stringPrinting.c [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+uint16_t GetSDLWhiteColor(SDL_Surface* sdl_screen)
+{
+	return GetSDLColor(sdl_screen, 255, 255, 255);
+}
+
+uint16_t GetSDLDarkGrayColor(SDL_Surface* sdl_screen)
+{
+	return GetSDLColor(sdl_screen, 70, 70, 70);
+}
```
Comment (Aemiii91) [inline src/clock/stringPrinting.c:22]: I would just have declared the colors statically:
```c
static SDL_Color color_white = {255, 255, 255};
static SDL_Color color_gray = {70, 70, 70};
```

So much boilerplate!

File: static/build/App/Clock/config.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (Aemiii91) [inline static/build/App/Clock/config.json:1]: This whole directory `static/build/App/Clock` should be moved to `static/packages/App/Set emulated time/App/Clock`

**Review by Aemiii91 — CHANGES_REQUESTED** [review]
I need the comments I've made resolved before merging this

File: static/build/App/Clock/config.json (hunk 3 above)
Comment (introkun) [reply static/build/App/Clock/config.json]: done

File: src/clock/stringPrinting.c (hunk 2 above)
Comment (introkun) [reply src/clock/stringPrinting.c]: done

File: Makefile (hunk 1 above)
Comment (introkun) [reply Makefile]: done

**Review by Aemiii91 — APPROVED** [review]

**Pull request merged**